packages/contract-wrappers/src/contract_wrappers/exchange_wrapper.ts

```diff
@@ -1097,6 +1098,11 @@ export class ExchangeWrapper extends ContractWrapper {
         const zrxAssetData = assetDataUtils.encodeERC20AssetData(zrxTokenAddress);
         return zrxAssetData;
     }
+    public async executeTransactionEncoderAsync(): Promise<ExecuteTransactionEncoder> {
```

Comment:
The name `executeTransactionEncoderAsync` was confusing to me since the first word is an verb that could also allude to the action this method does.

Perhaps `transactionEncoderAsync` is sufficient. This method should also have a comment explaining that this does not refer to an Ethereum tx.

Then the class would be `TransactionEncoder` and each method would be: `encodeFillOrderTx`.

Thoughts on this naming?